src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/TableSaw.java

```diff
@@ -65,6 +66,16 @@ null, new ItemStack(Material.IRON_BLOCK), null
             displayedRecipes.add(new ItemStack(plank));
             displayedRecipes.add(new ItemStack(Material.STICK, 4));
         }
+
+        if (Slimefun.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_20)) {
+            // Bamboo blocks to Bamboo planks
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_BLOCK));
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_PLANKS, 4));
+
+            // Stripped Bamboo Blocks to Bamboo planks
+            displayedRecipes.add(new ItemStack(Material.STRIPPED_BAMBOO_BLOCK));
+            displayedRecipes.add(new ItemStack(Material.BAMBOO_PLANKS, 4));
+        }
```

Comment:
Nitpick and totally unnecessary change but maybe put this between the log to plank and plank to stick, for consistency